tools/eubfr-cli/bin/cli.js

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env node
+
+const program = require('commander');
+const readline = require('readline');
+const pkg = require('../package.json');
+
+// Utilities
+const ensureVariables = require('../lib/ensureVariables');
+const getCredentials = require('../lib/getProducerCredentials');
+const getEndpoints = require('../lib/getEndpoints');
+const getAllProducers = require('../lib/getAllProducers');
+
+// Commands
+const uploadFiles = require('../commands/upload');
+const showFile = require('../commands/show');
+const deleteFiles = require('../commands/delete');
+
+// If -p is passed without actual value, it will be boolean true
+// Which is useless information in our case
+const hasValidProducer = options =>
+  options.producer && typeof options.producer !== 'boolean';
+
+const endpoints = getEndpoints();
```

Comment:
it should maybe be wrapped in a try/catch block (in case the `.env` files don't exist)

Reply:
If there are no .env files and this function does not manage to gather information it will return `{}` and the ensureVariables will warn and exit for the first missing required environment variable.